lib/srp.js

```diff
@@ -0,0 +1,273 @@
+import { BigNumber, Modulus } from 'asmcrypto.js/asmcrypto.all.es8';
```

Comment:
```suggestion
import { BigNumber, Modulus } from 'asmcrypto.js/dist_e8/bignum/bignum';
```
Not sure how well tree shaking works with the `all.es8` version but since they export a separate file anyway it might be good to use that.
Also, openpgp.js indirectly imports this file as well (but doesn't export it). At some point in the future we should probably include openpgpjs directly in pmcrypto, so that we can deduplicate that, have two builds of pmcrypto (compat and non compat), and then we also don't need to wait for `init(openpgp)` here anymore.